listenstore/listenstore/listenstore.py

```diff
@@ -242,6 +241,7 @@ def fetch_listens_for_range(self, uid, date_range, from_id, to_id, order, limit=
 
             self.log.debug("Fetching up to %s rows for date_range: %s, uid: %s", this_limit, date_range, uid)
             results = self.execute(query, params)
+            output = []
```

Comment:
This is unused. I would also suggest to remove other unrelated changes in this file or move them to a separate commit/pull request.